pypi_formula_mappings.json

```diff
@@ -230,6 +230,9 @@
   "nox": {
     "exclude_packages": ["six"]
   },
+  "nvchecker": {
+    "extra_packages": ["packaging"]
+  },
```

Comment:
Probably better to enable feature directly.
```suggestion
  "nvchecker": "nvchecker[pypi]",
```

Reply:
Thanks, it looks better. Why put `nvchecker[pypi]` here? Is there documentation?